packages/orchestrator-ui-components/src/hooks/useOrchestratorTheme.ts

```diff
@@ -8,9 +8,12 @@ export const useOrchestratorTheme = () => {
 
     const toSecondaryColor = (color: string) => tint(color, 0.8);
 
+    const toStatusColorFieldColor = (color: string) => tint(color, 0.3);
```

Comment:
In contrast with toSecondaryColor, this one is not theme specific. The color palette provided by the designer specifies a secondary color. I would suggest to move `toStatusColorFieldColor` closer to the service ticket components.

Reply:
I was asked to make the colors for the colorColumns lighter than the primary, but the secondary was way too light. I can move it to WfoColorColumn, because it's not going to be a ServiceTicket specific thing.